public/assets/js/main.js

```diff
@@ -0,0 +1,109 @@
+// Banner Carousel
+$(document).ready(function () {
+  $('.bxslider').bxSlider({
+    mode: 'fade',
+    auto: 'true'
+  });
+});
+
+// Login & register
+function validateLogin() {
+  let email = document.getElementById("email").value;
+  let password = document.getElementById("password").value;
+  if (email == "sabka@gmail.com" && password == "sabkabazar") {
```

Comment:
const

Reply:
check the authenticity of the input credentials through a function call. Function should exist in a different util or helper file